Resource based sampling GA docs update (#25806)
DataDog/documentation: content/en/tracing/trace_pipeline/ingestion_controls.md

Ingestion controls affect what traces are sent by your applications to Datadog. [APM Metrics][1] are always calculated based on all traces, and are not impacted by ingestion controls.

The Ingestion Control page provides visibility into the ingestion configuration of your applications and services. From the [ingestion control page][2]:

- Gain visibility on your service-level ingestion configuration.
- Adjust trace sampling rates for high throughput services or endpoints to better manage ingestion budget.
- Adjust trace sampling rates for low throughput, rare traffic services or endpoints to increase visibility.
- Understand which [ingestion mechanisms][11] are responsible for sampling most of your traces.
- Investigate and act on potential ingestion configuration issues, such as limited CPU or RAM resources for the Agent.

{{< img src="tracing/trace_indexing_and_ingestion/ingestion_control_page.png" style="width:100%;" alt="Ingestion Control Page Overview" >}}

## Understanding your ingestion configuration

> Use the three numbers in the ingestion control page header to get an overview of the total ingested data over the past hour, and an estimation of your monthly usage against your monthly allocation, calculated with the active APM infrastructure (hosts, Fargate tasks, and serverless functions).

Reviewer: ```suggestion
Use the data in the ingestion control header to monitor your trace ingestion. The header displays the total amount of data ingested over the past hour, your estimated monthly usage, and the percentage of your allocated monthly ingestion limit, calculated based on your active APM infrastructure (such as hosts, Fargate tasks, and serverless functions).
```

: Average number of traces per second ingested starting from the service over the past one hour.

Ingested Bytes/s
: Average number of bytes per second ingested for the service over the past one hour.

Downstream Bytes/s
: Average number of bytes per second ingested for which the service _makes the sampling decision_. This includes the bytes of all downstream services' spans in the call stack that follow the decision made at the head of the trace. This column's data is based on the `sampling_service` dimension, set on the `datadog.estimated_usage.apm.ingested_bytes` metrics. For more information, read [APM usage metrics][15].

Traffic Breakdown
: A detailed breakdown of traffic sampled and unsampled for traces starting from the service. See [Traffic breakdown](#traffic-breakdown) for more information.

Ingestion Configuration

> : Shows `Automatic` if the [default head-based sampling mechanism][4] from the Agent applies. If the ingestion was configured with [trace sampling rules][8], the service is marked as `Configured`; a `Local` label is set when the sampling rule is applied via configuration in the tracing library, a `Remote` label is set when the sampling rule is applied remotely, from the UI. For more information about configuring ingestion for a service, read about [changing the default ingestion rate](#configure-the-service-ingestion-rate).

Reviewer: ```suggestion
: Shows `Automatic` if the [default head-based sampling mechanism][4] from the Agent applies. If the ingestion was configured with [trace sampling rules][8], the service is marked as `Configured`; a `Local` label is set when the sampling rule is applied from configuration in the tracing library, a `Remote` label is set when the sampling rule is applied remotely, from the UI. For more information about configuring ingestion for a service, read about [changing the default ingestion rate](#configure-the-service-ingestion-rate).
```

**Note**: The table is powered by the [usage metrics][10] `datadog.estimated_usage.apm.ingested_spans` and `datadog.estimated_usage.apm.ingested_bytes`. These metrics are tagged by `service`, `env` and `ingestion_reason`.

#### Traffic breakdown

The Traffic Breakdown column breaks down the destination of all traces starting from the service. It gives you an estimate of the share of traffic that is ingested and dropped, and for which reasons.

{{< img src="tracing/trace_indexing_and_ingestion/service_traffic_breakdown.png" style="width:100%;" alt="Traffic breakdown of trace ingestion" >}}

The breakdown is composed of the following parts:

- **Complete traces ingested** (blue): The percentage of traces that have been ingested by Datadog.
- **Complete traces not retained** (gray): The percentage of traces that have not been ingested by Datadog. Some traces might be dropped because: 

    1. By default, the [Agent automatically sets a sampling rate][4] on services, depending on service traffic.

>     2. The service is configured to ingest a certain percentage of traces via [sampling rules][8].

Reviewer: ```suggestion
    2. The service is configured to ingest a certain percentage of traces using [sampling rules][8].
```

- **Complete traces dropped by the tracer rate limiter** (orange): When you choose to manually set the service ingestion rate as a percentage with trace sampling rules, a rate limiter is automatically enabled, set to 100 traces per second by default. See the [rate limiter][8] documentation to change this rate.

- **Traces dropped due to the Agent CPU or RAM limit** (red): This mechanism may drop spans and create incomplete traces. To fix this, increase the CPU and memory allocation for the infrastructure that the Agent runs on.

## Configuring ingestion for a service

> Click on any service to view the Service Ingestion Summary, a detailed view providing actionable insights on the ingestion configuration of the service.

Reviewer: ```suggestion
Click on any service to view the Service Ingestion Summary, which provides actionable insights and configuration options for managing that service's trace ingestion.
```

2. Click **Add new rule** to set sampling rates for some resources. Sampling rules use glob pattern matching, so you can use wildcards (`*`) to match against multiple resources at the same time.

#### if the Remote configuration option is available

3. Click **Apply** to save the configuration. 

The configuration should take effect in less than a minute. You **do not need** to redeploy the service for the change to take effect. You can observe the configuration changes from the [Live Search Explorer][9].

From the **Service Ingestion Summary**, resources for which the sampling rate are remotely applied should show as `Remote Configured` in the **Configuration** column.

#### if the Remote configuration option is disabled

If the remote configuration option is disabled, make sure that the listed [requirements](#remote-configuration-requirements) are all met to be able to use remote configuration.

3. Apply the appropriate configuration generated from these choices to the indicated service and redeploy the service. 
**Note**: The service name value is case sensitive. It should match the case of your service name.

> 4. Confirm on the Ingestion Control Page that your new percentage has been applied by looking at the Traffic Breakdown column, which surfaces the sampling rate applied. The resources for which the sampling rate was applied should show as `Local Configured`.

Reviewer: I think it would be easier to read if we put each of these paths into a tab or collapse-content shortcode. Let me know if you'd like help applying that!
